ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Deploying an operator in the bundle format using operator-sdk run bundle command"
+
```

Comment:
`operator-sdk` only supports certain install modes, so it would be useful if the step validated the parameter contains one of the supported falues and failed fast with a reasonable message if it does not.

Reply:
Fixed it.

Reply:
Fixed it.